src/kOS/Function/PrintList.cs

```diff
@@ -189,8 +189,10 @@ private kList GetResourceList(SharedObjects shared)
             foreach (Part part in shared.Vessel.Parts)
             {
                 string stageStr = part.inverseStage.ToString();
-                foreach (PartResource resource in part.Resources)
+                PartResource resource;
+                for (int i = 0; i < part.Resources.Count; ++i)
```

Comment:
I actually prefer the loop indexing myself because iterators seem like unneeded overhead most of the time.  I was just wondering if the change was required because the iterator no longer existed, or if it was done just for performance reasons.